forge-gui/res/cardsfolder/upcoming/contested_bame_ball.txt

```diff
@@ -0,0 +1,12 @@
+Name:Contested Game Ball
+ManaCost:2
+Types:Artifact
+T:Mode$ DamageDoneOnce | CombatDamage$ True | ValidTarget$ You | ValidSource$ Card | TriggerZones$ Battlefield | Execute$ TrigGainGontrol | TriggerDescription$ Whenever you're dealt combat damage, the attacking player gains control of CARDNAME and untaps it.
+SVar:TrigGainGontrol:DB$ GainControl | NewController$ TriggeredAttackingPlayer | Untap$ True
+A:AB$ Draw | Cost$ 2 T | SubAbility$ DBPutCounter | SpellDescription$ Draw a card and put a point counter on CARDNAME. Then if it has five or more point counters on it, sacrifice it and create a Treasure token.
+SVar:DBPutCounter:DB$ PutCounter | CounterType$ POINT | CounterNum$ 1 | SubAbility$ DBSac
+SVar:DBSac:DB$ Sacrifice | SacValid$ Self | ConditionPresent$ Card.Self+counters_GE5_POINT | RememberSacrificed$ True |  SubAbility$ DBTreasure
+SVar:DBTreasure:DB$ Token | TokenScript$ c_a_treasure_sac | ConditionDefined$ Remembered | ConditionPresent$ Card | SubAbility$ DBCleanup
```

Comment:
better use Branch

Reply:
Hmm can you do a branch when there's only 1 possible result (nothing should happen when it has 4 or less point counters) ?

Reply:
sure

Reply:
checkout treasure map:
https://github.com/Card-Forge/forge/blob/e1ea01f06584a9b42c4422387b6735b979b33ceb/forge-gui/res/cardsfolder/t/treasure_map_treasure_cove.txt#L6-L9

Reply:
Thanks will do

Reply:
Also for the Ball, doesn't say "if you do" for the Sacrifice.

so even if you can't sacrifice it, it should still make the Token.

so no RememberSacrificed needed